Meta/Lagom/CMakeLists.txt

```diff
@@ -27,6 +27,12 @@ set(SerenityOS_SOURCE_DIR "${SERENITY_PROJECT_ROOT}" CACHE STRING "")
 
 set(SERENITY_CACHE_DIR "${PROJECT_BINARY_DIR}" CACHE STRING "")
 
+# Place binaries in a staging directory to ensure that rpaths always
+# remain valid.
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${SERENITY_PROJECT_ROOT}/Build/lagom/staging/bin")
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${SERENITY_PROJECT_ROOT}/Build/lagom/staging/${CMAKE_INSTALL_LIBDIR}")
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${SERENITY_PROJECT_ROOT}/Build/lagom/staging/${CMAKE_INSTALL_LIBDIR}")
```

Comment:
This breaks a lot of things. There's CI scripts, local scripts in Meta/, serenity.sh, etc that assume paths to Lagom binaries in the build directory.

Reply:
Hmm. well, this is the approach we use for jakt as well, and it seems to work there. so I think if we fixed all the places that have assumptions about build path to binaries, then this approach would be fine. Not sure about the `staging` directory though. That might be harder than it seems though.